README.md

```diff
@@ -85,15 +85,15 @@ func showDropIn(clientTokenOrTokenizationKey: String) {
 
 ### Apple Pay + Drop-In
 
-Apple Pay is enabled by default in Drop-In. Unless you opt out, by setting `showApplePayPaymentOption = false`, Drop-In will show Apple Pay as a payment option as long as it is enabled in the control panel. Below is an example of hiding the Apple Pay button if the device can't make Apple Pay payments using certain card networks:
+Apple Pay is enabled by default in Drop-In. Unless you opt out, by setting `applePayDisabled = true`, Drop-In will show Apple Pay as a payment option as long as it is enabled in the control panel. Below is an example of hiding the Apple Pay button if the device can't make Apple Pay payments using certain card networks:
```

Comment:
This isn't really related to the other changes in this PR -- I just happened to notice that this variable name was wrong.